packages/ui-causal-graph-editor/src/shared/rendering/grouping/group-container-object.tsx

```diff
@@ -109,22 +109,19 @@ export class GroupContainerObject extends PIXI.EventEmitter<(typeof MOUSE_EVENTS
         (groupContainerGfx.hitArea as PIXI.Rectangle).height = height;
 
         // Get/create rectangle texture
-        const rectangleTexture = textureCache.get(createKey(GROUP_RECTANGLE, minX, maxX, minY, maxY), () => {
+        const rectangleTexture = textureCache.get(createKey(GROUP_RECTANGLE), () => {
```

Comment:
This change fixes a memory leak. Previously this would create a new texture for each unique rectangle position.

Now there is no border at all so the rectangle texture is static and is then resized by the surrounding sprite which is much more performant.

Reply:
Thank you!